### DISCUSSION TITLE
What is the "recommended" way of defining method "templates"?

### DISCUSSION THREAD

**Discussion by SCLeoX** [eslint/eslint#14290] [Q&A]
Say I have:

```javascript
class Potion {
  onDrink(user) {}
}
class StrengthPotion extends Potion {
  onDrink(user) {
    user.strength += 10000000;
  }
}
```

Using recommended configurations, eslint currently reports `unused-var` for the `user` variable in the method `onDrink` of `Potion`. I am wondering, what is the intended way of doing something like this.

**Comment by nzakas** [comment]
We don’t have any intended patterns. If you’re not using an argument, you can either delete it or use a comment to disable the warning on that line.
